packages/lesswrong/components/comments/CommentsNewForm.tsx

```diff
@@ -284,8 +298,51 @@ const CommentsNewForm = ({
     if (formDisabledDueToRateLimit) {
       submitBtnProps.disabled = true
     }
-    
+
+    const { DialogueCheckBox } = Components
+
+    const targetUserId = parentComment?.userId
+    const targetUserDisplayName = parentComment?.user?.displayName
+
+    const {
+      userDialogueChecksResult: { results: userDialogueChecks = [] },
+    } = useDialogueMatchmaking({ getMatchedUsers: false, getRecommendedUsers: false, getOptedInUsers: false, getUserDialogueChecks: true });
+
+    const { checkId, userIsChecked, userIsMatched } = parentComment?.user ? getUserCheckInfo(parentComment?.user, userDialogueChecks) : { checkId: null, userIsChecked: null, userIsMatched: null }
+    console.log(document, targetUserId, targetUserDisplayName, checkId, userIsChecked, userIsMatched)
+
+    const { LWHelpIcon, LWTooltip } = Components
+
+    const test = useABTest(showMatchBoxesInCommentReplyFormABTest)
```

Comment:
This variable name needs to be more specific

Reply:
I've called it "testGroup", if you'd like something else please provide a name and I will change it